### DISCUSSION TITLE
Anchor Problem

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#814] [Q&A]
Hi, why I cannot see the Anchor status box when I already connect the power to loco position node ? I follow the bitcraze instruction and set the anchor number and mode. 
![1A8796C4-3309-44AC-88D2-F155A06CCFD2_1_102_o](https://github.com/bitcraze/discussions/assets/118436188/741c9ba0-dbb1-493a-a074-72e2a76c3304)

**Comment by ataffanel** [comment]
If looks like the Crazyflie is not receiving signals from the anchors.

Are you anchors ID and modes already configured using the PC tools? Does the mode on the Crazyflie side manages to settle on the anchor mode?

**Reply by CalvinKoung** [reply]
Now, I can see the anchor appear the green box in the screen, but I cannot see the position. I added the anchor by myself but nothing happen. 
![A6E6A722-024A-4660-9DA7-1D22CEE31E3C](https://github.com/bitcraze/discussions/assets/118436188/a79b4375-896a-4b55-8c91-e1adb9de6785)

**Reply by ataffanel** [reply]
Good that you have them in! You now need to manually enter the position to match the configuration of the anchors in your space.

**Reply by CalvinKoung** [reply]
hi, I am a new question. the blue circle is crazyflie but when I move the crazyflie to another anchor location. I cannot see the blue circle move. What reason lead to this problem.

**Reply by ataffanel** [reply]
Have you setup the position for all the anchors? If so, the Crazyflie position should be estimated and the estimation will be best within the convex-hull formed by the anchors. Can you send a screenshot of your setup with anchor position?
